src/context/AuthContext/AuthContext.jsx

```diff
@@ -0,0 +1,34 @@
+import { createContext, useContext, useState, useEffect } from 'react';
+import { useAxios } from '../../CustomHooks';
+
+const AuthContext = createContext();
+
+const AuthProvider = ({ children }) => {
+	const [user, setUser] = useState(null);
```

Comment:
I think a better approach would be to have a single state for the user info. For both the encoded token and user details.

Reply:
removed